playground/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["*.ts", "**/__test/*.ts", "**/vite.config.ts"],
```

Comment:
I think this was a typo?
My IDE (vscode) wasn't recognizing `~utils` in `playground/ssr-react/__tests__/ssr-react.spec.ts`. I assume it was becuase of this.